packages/components/Badge/SPEC.md

```diff
@@ -267,6 +267,8 @@ Presence badge represents user's availability.
 
 ### PresenceBadge Props
```

Comment:
Query unrelated to your change - is `PresenceBadge` exported as a component for it's used only within library for avatar?